docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4762,6 +4765,9 @@ Initialize a new swarm. The body of the HTTP response includes the node ID.
         "Raft": {},
         "Dispatcher": {},
         "CAConfig": {}
+        "EncryptionConfig" : {
+          "AutoLockManagers": false
+        },
```

Comment:
for a follow up; remove the trailing comma, and add a comma after `"CAConfig": {}`